custom-words.txt

```diff
@@ -3977,3 +3977,16 @@ protobuf
 proto
 msvideo
 SOQL
+webappcollectors
+wacc
+hyperthread
+Hyperthread
+Datacentre
+Capex
+capex
+Opex
+opex
+decomission
+Decommisioned
+Decommision
+EUAP
```

Comment:
please remove EUAP now that regions are not an enum

Reply:
Done